pkg/controller/topology.go

```diff
@@ -281,23 +246,7 @@ func getSelectedCSINode(
 	selectedNode *v1.Node) (*storagev1.CSINode, error) {
 
 	selectedCSINode, err := csiNodeLister.Get(selectedNode.Name)
-	if err != nil {
-		// If the Node is before 1.14, then we fallback to "topology disabled" behavior
-		// to retain backwards compatibility in a single-topology environment with
-		// delayed binding.
-		//
-		// TODO (#257): Once K8s 1.13 is no longer supported, then this check can be removed
-		// and we can require that CSINode exists.
-		nodeVersion, parseErr := version.ParseSemantic(selectedNode.Status.NodeInfo.KubeletVersion)
-		if parseErr != nil {
-			return nil, fmt.Errorf("Failed to parse kubelet version from node %q: %v", selectedNode.Name, parseErr)
-		}
-		if nodeVersion.LessThan(k8sTopologyBetaVersion) {
-			klog.Warningf("Selected node %q version %q is less than %q, falling back to no topology", selectedNode.Name, nodeVersion, k8sTopologyBetaVersion)
-			return nil, nil
-		}
-
-		// Otherwise, require the CSINode beta feature on the K8s node to also be enabled.
+	if err != nil || selectedNode == nil {
```

Comment:
`selectedCSINode`

The error message should be different if `selectedCSINode` is nil - `err` might be nil

Reply:
done